README.md

```diff
@@ -1 +1,199 @@
-# CAPSTONE-1
\ No newline at end of file
+# Confrence Website(CAPSTONE-1)
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Portfolio] <a name="Confrence website"></a>
+
+**[Portfolio]** This is a simple Summit Confrence site, that is simple and yet elegant,classic and unique.
+
+## 🛠 Built With <a name="built-with"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">Css</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+- [ ] **[Hero-section]**
+- [ ] **[Program-section]**
+- [ ] **[About-page]**
+- [ ] **[Speakers-Section]**
+- [ ] **[Mobile-view]**
+- [ ] **[Desktop-view]**
+- [ ] **[Dynamic-js-Display]**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://shabiblac.github.io/Microverse-Portfolio-Design/)
+
+
```

Comment:
- [ ] Great job working on the deployement I think you deployed it and it is working fine but you forget to add it here sicne you put here the wrong link so kindly add the right link for this project 🙏🏻💪🏻🔥🤩